src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double height;
+
+    public Rectangle(String color, double length, double height) {
+        super(color);
+        this.length = length;
+        this.height = height;
+    }
+
+    public double calculateArea() {
```

Comment:
The method `calculateArea` should be marked with the `@Override` annotation since it's likely intended to override a method from the parent class `Figure`. This provides better readability and helps to avoid errors.